Grievance_Backend/grievance/src/main/java/com/grievance/service/TicketServiceImpl.java

```diff
@@ -298,29 +308,159 @@ public Optional<List<TicketOutWOComment>> listTicketsByStatus(
     return Optional.ofNullable(list);
   }
 
+  /**
+  *
+  * @param departmentId
+  * @param status
+  * @param page
+  * @return list of tickets by department and status
+  */
+  @Override
+  public Optional<List<TicketOutWOComment>> listTicketsByDepartmentAndStatus(
+      final Integer departmentId,
+      final Status status,
+      final Integer page) {
+    LOGGER.info("Listing tickets by department {}, "
```

Comment:
Put loggers at controller level, no need in service layer

Reply:
But there're method that are calling different services so which service method is being called are specified by Controller.